WordPress/wordcamp.org
# Blocks: Update packages

Fixes #257 – Tests are failing because we're using node 12+, which needs package updates for compatibility. I've take this time to update _all_ the packages, and fix resulting issues.

- Update how asset manifests are generated and used for script dependencies
- Fix issue with importing stripTags, which is not actually a `@wordpress/*` package

**To test**

- Run `nvm install` to install the latest `lts/*` version
- Run `nvm use` to officially switch to the `lts/*` version
- Run `npm run build` to make sure it builds, test out a few blocks
- Make sure this passes in travis
